changelog.d/8971-key-scan-kills-read-lane.md

```
@@ -0,0 +1,36 @@
+The last four per-element key scans on the property paths now resolve through
+the shape hash index.
+
+#8936 and #8950 replaced the `js_array_get` + `js_string_key_matches` walks on
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)